org.eclipse.lyo.oslc4j.codegenerator/src/org/eclipse/lyo/oslc4j/codegenerator/files/generateAdaptorServletListener.mtl

```diff
@@ -89,15 +85,17 @@ public class [javaClassNameForAdaptorServletListener(anAdaptorInterface) /] impl
     public void contextInitialized(final ServletContextEvent servletContextEvent)
     {
         //These are default values. You can modify any of them early in this method.
-        String basePathEnvKey = "LYO_BASE";
-        String basePathContextPropertyKey = String.format("%s.%s", [javaClassNameForAdaptorServletListener(anAdaptorInterface) /].class.getPackage().getName(), "baseurl");
+        String basePathKey = "baseurl";
```

Comment:
We should write in the changelog that `LYO_BASE` will be renamed to `LYO_BASEURL` to maintain consistency of all property setting mechanism.

@jadelkhoury I think this the perfect time to add CHANGELOG.md to LyoD